src/Components/Home/About/About.scss

```diff
@@ -1,29 +1,151 @@
-.contain {
+@import "../../../scss/imports";
+@import "../../../scss/base";
```

Comment:
You don't need to import `base`. If you have added anything to base which you would like to import, add it to a file in imports instead of base.

Reply:
Done sir